repos/system_upgrade/el7toel8/files/rhel_upgrade.py

```diff
@@ -62,6 +62,8 @@ def configure(self):
         for repo in self.base.repos.all():
             if repo.id in enabled_repos:
                 repo.skip_if_unavailable = False
+                if not self.base.conf.gpgcheck:
+                    repo.gpgcheck = False
```

Comment:
Why not simply `repo.gpgcheck = self.base.conf.gpgcheck`?

Reply:
Because I don't want it to be always the same, I just want to disable it, if it is enabled. And I want to make it explicit in the code.

Reply:
Ok. So if the dnf plugin data tell to *do* gpg checking, we'll leave it up to the setup of each repo. Which is the default behaviour of dnf I guess. Fair point.